x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx

```diff
@@ -30,19 +31,26 @@ const TransactionBreakdownGraph: React.FC<Props> = props => {
       return {
         title: timeseriesConfig.name,
         color: timeseriesConfig.color,
-        data: timeseriesConfig.values,
+        data: timeseriesConfig.values.map(value => {
+          return 'y' in value && isValidCoordinateValue(value.y)
+            ? value
+            : {
+                ...value,
+                y: undefined
+              };
+        }),
```

Comment:
I'm struggling a bit following the flow due to the many transformations this takes. It seems this is the three places where breakdown graphs are being transformed:

https://github.com/elastic/kibana/blob/7ad2c947b3d380a967fc1fbea33dec2bd5297152/x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.ts#L189-L192

https://github.com/elastic/kibana/blob/7ad2c947b3d380a967fc1fbea33dec2bd5297152/x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/index.tsx#L87-L100

https://github.com/elastic/kibana/blob/7ad2c947b3d380a967fc1fbea33dec2bd5297152/x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx#L29-L38

Would it be possible for you to combine all (or most) of the transformations and do it on the backend?